### ISSUE TITLE
Custom Furniture Mod Error

### ISSUE THREAD

**Issue by wally232** [ZaneYork/SMAPI-Android-Installer#48]
Custom furniture
It cannot be recalled while the mode is in use.
You can install furniture in this mode, but ...
After installing the furniture, if you save
Can't load.
I wonder if this mod is not available on mobile.

**Comment by ZaneYork** [comment]
Yes, this is the known issue introduced by  CFR, use PFM instead

**Comment by wally232** [comment]
Is CFR mod Custom Farming Redux mod?

Custom furniture
What are the two relationships?

Let me tell you about the furniture installation mode..


Are you talking about the Producer Framework Mod regarding PEM?

Are the above mods related to furniture installation?

**Comment by ZaneYork** [comment]
They are both depends on PyTK, they have the same issue of placing item onto any scene. But the Producer Framework Mod won't, so the resource pack depends on PFM works fine on mobile

**Comment by wally232** [comment]
https://www.nexusmods.com/stardewvalley/mods/1254
There is a problem with the above mods.
Custom Furniture mode..

I'm not talking about the PFM mode...
